datafusion/core/src/test/mod.rs

```diff
@@ -359,15 +297,88 @@ pub fn csv_exec_sorted(
     ))
 }
 
-/// Create batch
-fn create_batch(schema: &Schema) -> RecordBatch {
-    RecordBatch::try_new(
-        Arc::new(schema.clone()),
-        vec![Arc::new(UInt32Array::from(vec![1, 2, 3, 4, 5, 6, 7, 8]))],
-    )
-    .unwrap()
+/// A mock execution plan that simply returns the provided statistics
+#[derive(Debug, Clone)]
+pub struct StatisticsExec {
```

Comment:
I couldn't find where this was moved from

Reply:
I am not sure -- there are defiitely some areas that would be nice to clean up with respect to testing